packages/manager/apps/dedicated/client/app/config/constants.config.js

```diff
@@ -758,7 +758,7 @@ const constants = {
   CA: {
     RENEW_URL:
       'https://ca.ovh.com/cgi-bin/order/renew.cgi?domainChooser={serviceName}',
-    vmsUrl: 'http://status.ovh.net/vms/',
+    vmsUrl: 'http://vms.status-ovhcloud.com/',
```

Comment:
The address does not exist in secure with https?

Reply:
yes doesn't exist unfortunately